test/fixtures/e2e/amp-story-player/pre-rendering.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Story Player (Non-AMP)</title>
+    <link href="../../../../build/css/amp-story-player-v0.css" rel='stylesheet' type='text/css'>
+    <script async src="../../../../dist/amp-story-player.js"></script>
+    <style page>
+      body {
+        font-family: sans-serif;
+      }
+    </style>
+    <style injected>
+      .fake-content {
+        position: relative;
+        height: 10000px;
+      }
+    </style>
```

Comment:
Why do you need to `style` tags?

Reply:
We need to create a gap to make the player be far below the current viewport so we can test our preloading strategies when the player isn't close to the viewport.